app/build.gradle

```diff
@@ -13,7 +12,7 @@ android {
         applicationId "dev.arkbuilders.rate"
         minSdk 23
         targetSdk 34
-        versionCode 2
+        versionCode 3
```

Comment:
Is this version change on purpose? 
If yes, why not `versionName` changed too?

Reply:
There is no need for the versionName to change with the versionCode simultaneously.

Reply:
@MohamedPresident So what is the purpose of increasing versionCode, if it's not for production release? If it's just for testing purpose, I don't think it's required.

Reply:
No, it is required. just do Internal testing for the first time and you will find it is required.